src/main/java/lotto/domain/Price.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class Price {
+
+    private final int value;
+
+    public Price(int value) {
+        validate(value);
+        this.value = value;
+    }
+
+    public int getValue() {
+        return this.value;
+    }
+
+    public double profitPercent(int totalResultPrice) {
+        return (double) totalResultPrice / this.value;
+    }
+
+    private void validate(int value) {
+        checkMinPrice(value);
+        checkDivisibleByThousand(value);
+    }
+
+    private void checkMinPrice(int value) {
+        if (value < 1000) {
```

Comment:
```suggestion
        if (value < 1_000) {
```
돈을 나타내는 숫자의 경우 천 단위를 구분하기 위해 "1_000"와 같이 언더바를 통해 구분할 수 있다.